src/connectivity.jl

```diff
@@ -748,3 +760,73 @@ function isgraphical(degs::Vector{<:Integer})
     end
     return true
 end
+
+"""
+    isdigraphical(indegree_sequence, outdegree_sequence)
+
+Check whether the given indegree sequence and outdegree sequence are digraphical, that is whether they can be the indegree and outdegree sequence of a digraph.
+
```

Comment:
We should probably mention here that
* The digraph cannot have self-loops
* `indegree_sequence` and `outdegree_sequence`  are not independent, i.e. if entry `i` of `indegree_sequence`  has value `a_i` and entry `i` of `outdegree_sequence` has value `b_i` then there will be a vertex with indegree `a_i` and outdegree `b_i` in the realization. (maybe there is an easier way to express this).

Reply:
We modified the docstring trying to include your suggestions. Thanks.